db/migrate/20220817075919_add_last_submitted_at_to_exercise_representations.rb

```diff
@@ -0,0 +1,11 @@
+class AddLastSubmittedAtToExerciseRepresentations < ActiveRecord::Migration[7.0]
+  def change
+    add_column :exercise_representations, :last_submitted_at, :datetime, null: false, default: Time.now, if_not_exists: true
+
+    unless Rails.env.production?
+      Exercise::Representation.find_each do |rep|
+        rep.update(last_submitted_at: rep.submission_representations.last.created_at)
+      end
```

Comment:
Can you put together a bit on the OP of this PR with all the things we need to run before we deploy pls.